build/docker/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.sql.gz
```

Comment:
Can we move this tow SQL ignore lines to `build/docker/data/assets/sql/.gitignore` file?